vidarrtest-regression.json.in

```diff
@@ -58,10 +58,10 @@
                         },
                         "type": "EXTERNAL"
                     },
-                    "readGroup": "ID:121005_h804_0096_AD0V4NACXX_4_NoIndex PL:Illumina PU:121005_h804_0096_AD0V4NACXX_4_NoIndex LB:K562_1 SM:K562_1test CM:OICR"
+                    "readGroup": "ID:Test_NoIndex PL:Illumina PU:Test_NoIndex LB:K_1 SM:K_1test CN:OICR"
                 }
             ],
-            "star.outputFileNamePrefix": "K562",
```

Comment:
K562 is the name of a cell line, it is not OICR-specific or sensitive informatio